src/vs/platform/environment/test/node/argv.test.ts

```diff
@@ -112,13 +112,13 @@ suite('parseArgs', () => {
 			_: string[];
 		}
 
-		const options1: OptionDescriptions<TestArgs1> = {
+		const options1 = {
 			'testcmd': c('A test command', {
 				testArg: o('A test command option'),
 				_: { type: 'string[]' }
 			}),
 			_: { type: 'string[]' }
-		};
+		} as OptionDescriptions<TestArgs1>;
```

Comment:
`satisfies` instead of `as` would type-check (`as` just type-casts). (Also below.)